pytripgui/main_window_qt_vc/main_window_view.py

```diff
@@ -38,6 +38,10 @@ def browse_file_path(self, name, extension, path=None):
         selected_file = QFileDialog.getOpenFileName(self.ui, name, path, extension)
         return selected_file[0]
 
+    def save_file_path(self, caption, extension, path=None):
+        file = QFileDialog.getSaveFileName(self.ui, caption, path, extension)
+        return file[0]
```

Comment:
Add a comment explaining why `file[0]` is returned, not simply `file`
Change `file` to something more meaningful - is it path to the file, file object or a filename ?

Reply:
Modified the name to selected_file_path and named the other member of the returned tuple.